README.md

```diff
@@ -124,6 +124,8 @@ You can use the `scripts/setup-sso-idp.sh` script to setup a "testing-idp" realm
 With this script you will get few regular users - test-user[01-10] and few users that will be added to dedicated-admins group - customer-admin[01-03].
 
 Prerequisites:
+- `oc` command available on your machine (`sudo dnf install oc` on Fedora)
```

Comment:
oc version in dnf registry could outdated, it's better to link to https://github.com/openshift/oc/releases